new-log-viewer/src/services/LogFileManager/utils.ts

```diff
@@ -11,60 +18,56 @@ import {getBasenameFromUrlOrDefault} from "../../utils/url";
 
 
 /**
- * Gets the log event number range [begin, end) of the page that starts at the given log event
- * index.
+ * Gets the data for the `PAGE_NUM` cursor.
  *
- * @param beginLogEventIdx
+ * @param pageNum
+ * @param eventPositionOnPage
  * @param numEvents
  * @param pageSize
- * @return An array:
- * - pageBeginLogEventNum
- * - pageEndLogEventNum
+ * @return Matching page and log event for the cursor (i.e. cursor data.)
```

Comment:
```suggestion
 * @return
```

In this case, I think the type of the return value (and its associated docstring) is more descriptive than any return value description we write here. Ideally, our linting settings would allow us to just specify the current description as the return value, but until they do, I think having an empty return value description is our best option.